src/index.html

```diff
@@ -5,9 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <script type="module" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule="true" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"></script>
   </head>
   <body>
+
     <h1>Product cards</h1>
+    <div class="conteiner">
+      <div class="card">
+        <div class="card__top">
+          <img 
+          class="card__img" 
+          src="../src/images/imac.jpeg" 
+          alt="iMac"
+          >
+        </div>
+            
+        <div class="card__bottom">
+          <div class="card__title">
+            <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <h3>Product code: 195434</h3>
+          </div>
+
+          <div class="rating">
+            <div class="rating__items">
+              <div class="rating__star">★</div>
+              <div class="rating__star">★</div>
+              <div class="rating__star">★</div>
+              <div class="rating__star">★</div>
+              <div class="rating__star5">★</div>
+            </div>
+
```

Comment:
you must reuse all styles and tags from stars task

Reply:
I missed this task